### PR TITLE
Update soft-opt-in-consent-setter/README.md

### PR DESCRIPTION
**Pull request by ripecosta**
## What does this change?
Adds more detail to the `subsWith5RetriesAlarm` alarm, mentioning the most frequent cause.

### CODE REVIEW

File: handlers/soft-opt-in-consent-setter/README.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ will provide more details regarding which of these is taking place.
...
 1. Check that the IDAPI endpoints are correct and online.
 1. Check the logs for the error code being returned by the endpoint and contact the Identity team to understand why
-   IDAPI was unable to set the user's Soft Opt-In consents.
+   IDAPI was unable to set the user's Soft Opt-In consents. Most of the time this will be a user who no longer has an 
+   Identity account, in which case no action is necessary.
 
 For all the above, after the underlying issue is resolved, the `Soft_Opt_in_Number_of_Attempts__c` fields needs to be
-reset to 0 for reach of the affected records. After that the lambda will pick up those record for processing again.
\ No newline at end of file
+reset to 0 for reach of the affected records. After that the lambda will pick up those record for processing again.
```
Comment (kelvin-chappell) [inline handlers/soft-opt-in-consent-setter/README.md:118]:
```suggestion
reset to 0 for each of the affected records. After that the lambda will pick up those records for processing again.
```

File: handlers/soft-opt-in-consent-setter/README.md [hunk 2]
```diff
@@ -111,7 +111,8 @@ will provide more details regarding which of these is taking place.
 
 1. Check that the IDAPI endpoints are correct and online.
 1. Check the logs for the error code being returned by the endpoint and contact the Identity team to understand why
-   IDAPI was unable to set the user's Soft Opt-In consents.
+   IDAPI was unable to set the user's Soft Opt-In consents. Most of the time this will be a user who no longer has an 
+   Identity account, in which case no action is necessary.
 
 For all the above, after the underlying issue is resolved, the `Soft_Opt_in_Number_of_Attempts__c` fields needs to be
```
Comment (kelvin-chappell) [inline handlers/soft-opt-in-consent-setter/README.md:117]: Perhaps specify which object the field is on: `SF_Subscription__c.Soft_Opt_in_Number_of_Attempts__c`.

File: handlers/soft-opt-in-consent-setter/README.md [hunk 3]
```diff
@@ -111,7 +111,8 @@ will provide more details regarding which of these is taking place.
 
 1. Check that the IDAPI endpoints are correct and online.
 1. Check the logs for the error code being returned by the endpoint and contact the Identity team to understand why
-   IDAPI was unable to set the user's Soft Opt-In consents.
+   IDAPI was unable to set the user's Soft Opt-In consents. Most of the time this will be a user who no longer has an 
+   Identity account, in which case no action is necessary.
```
Comment (kelvin-chappell) [inline handlers/soft-opt-in-consent-setter/README.md:115]: If a contact record has an Identity ID that doesn't exist, should we remove it from the record?

File: handlers/soft-opt-in-consent-setter/README.md [hunk 4]
```diff
@@ -111,7 +111,8 @@ will provide more details regarding which of these is taking place.
 
 1. Check that the IDAPI endpoints are correct and online.
 1. Check the logs for the error code being returned by the endpoint and contact the Identity team to understand why
-   IDAPI was unable to set the user's Soft Opt-In consents.
+   IDAPI was unable to set the user's Soft Opt-In consents. Most of the time this will be a user who no longer has an
```
Comment (kelvin-chappell) [inline handlers/soft-opt-in-consent-setter/README.md:114]: Would it be worth suggesting using the useradmin tool to find the Identity account, if missing accounts is the main problem?

File: handlers/soft-opt-in-consent-setter/README.md (hunk 3 above)
Comment (ripecosta) [reply handlers/soft-opt-in-consent-setter/README.md:115]: I think a larger conversation has be to be had about that. That seems logical to me but there might be reasons not to do it. I've not delved too deep into it. @david-pepper will likely know more about it.

File: handlers/soft-opt-in-consent-setter/README.md (hunk 4 above)
Comment (ripecosta) [reply handlers/soft-opt-in-consent-setter/README.md:114]: I thought only Identity had access to that tool. I've never used it.

File: handlers/soft-opt-in-consent-setter/README.md (hunk 2 above)
Comment (ripecosta) [reply handlers/soft-opt-in-consent-setter/README.md:117]: Done!

File: handlers/soft-opt-in-consent-setter/README.md (hunk 4 above)
Comment (ripecosta) [reply handlers/soft-opt-in-consent-setter/README.md:114]: Do you want to add a quick line about it?

Comment (kelvin-chappell) [reply handlers/soft-opt-in-consent-setter/README.md:114]: Perhaps something like: 
> You can use the [useradmin tool](https://useradmin.gutools.co.uk/) to find out if an Identity ID exists in the Identity system.  This is a tool that allows you to tie together identities with email addresses, subscriptions, physical addresses etc.  If the ID doesn't exist in the system, there's no need to take any further action.

**Review by kelvin-chappell — APPROVED** [review]
That's clearer now.  Thanks! 👍

File: handlers/soft-opt-in-consent-setter/README.md (hunk 4 above)
Comment (ripecosta) [reply handlers/soft-opt-in-consent-setter/README.md:114]: Done!

File: handlers/soft-opt-in-consent-setter/README.md (hunk 3 above)
Comment (kelvin-chappell) [reply handlers/soft-opt-in-consent-setter/README.md]: I can see removing bad data could open a can of worms.  Let's consider this point resolved.

**Pull request merged**